daos.patch

```diff
@@ -1,63 +1,33 @@
-From 5cc79a0e878c96dc38a1b0ae48fabeae346a7dec Mon Sep 17 00:00:00 2001
-From: Maureen Jean <mjean@boro-69.boro.hpdd.intel.com>
-Date: Wed, 15 Jul 2020 20:43:28 +0000
-Subject: [PATCH] daos testphdf5
+From 4aa40365623cf35643b262781f92578220c42bbb Mon Sep 17 00:00:00 2001
+From: Larry Knox <lrknox@hdfgroup.org>
+Date: Mon, 2 Mar 2020 17:03:12 -0600
+Subject: [PATCH] Commit 1.12.0 release changes to README and RELEASE.txt.
 
 ---
- testpar/testphdf5.c | 10 ++++++++++
- 1 file changed, 10 insertions(+)
+ README.txt               | 2 +-
+ release_docs/RELEASE.txt | 2 +-
+ 2 files changed, 2 insertions(+), 2 deletions(-)
 
-diff --git a/testpar/testphdf5.c b/testpar/testphdf5.c
```

Comment:
something seems muggled here.. it's actually removing all the changes to actually disabled the unsupported tests?

Reply:
I am not sure why the diffs appear like this but I believe the last commit to daos.patch is correct.  The patch adds the ifdef to the unsupported tests in the 1.12 version of testphdf5.c.    I did regenerate that patch because the 1.12 source file was a bit different than the 1.10 .c file.

Reply:
@mchaarawi Can you take a look at the patch again..   I now realize that your comment was correct and that I did push the wrong patch file and quickly updated it last night.  Please review the final patch to be sure I did not miss anything.  

I ran the daos PR to test out the 1.12 hdf5 rpms and it did pass with the latest daos.patch change.

https://github.com/daos-stack/daos/pull/3136